packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -114,15 +119,30 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
     };
   }, [meetingAdapter]);
 
-  const [showChat, setShowChat] = useState(false);
-  const [showPeople, setShowPeople] = useState(false);
+  useEffect(() => {
+    const incrementUnreadtChatMessagesCount = (event: { message: ChatMessage }): void => {
+      if (!showChat && event.message.senderDisplayName !== '') {
+        if (event.message.type === 'text' || event.message.type === 'html') {
+          setUnreadChatMessagesCount(unreadChatMessagesCount + 1);
+        }
+      }
+    };
+    meetingAdapter.on('messageReceived', incrementUnreadtChatMessagesCount);
+
+    return () => {
+      meetingAdapter.off('messageReceived', incrementUnreadtChatMessagesCount);
+    };
+  }, [meetingAdapter, setUnreadChatMessagesCount, showChat, unreadChatMessagesCount]);
 
   const closePane = useCallback(() => {
     setShowChat(false);
     setShowPeople(false);
   }, []);
 
   const toggleChat = useCallback(() => {
+    if (!showChat) {
```

Comment:
add a comment here so this is clear -- 
```suggestion
    // When the chat pane is opened, mark all chat messages as read
    if (!showChat) {
```